docs/monitoring/monitoring-troubleshooting.asciidoc

```diff
@@ -32,6 +32,10 @@ The *Monitoring* page in {kib} is empty.
 
 *Resolution:*
 
+. Confirm that you are not using {kib} 6.7.0 or 6.7.1. If you have a version 7.0
```

Comment:
Should we say that this is due to a bug and reference the GitHub issue here?

Reply:
++

Reply:
Done, thanks!